sync/gh.py

```diff
@@ -125,17 +125,14 @@ def approve_pull(self, pr_id):
             input=post_parameters
         )
 
-    def status_checks_pass(self, pr_id):
-        pr = self.get_pull(pr_id)
-        if not pr.mergeable:
-            return False
-        statuses = self.get_statuses(pr_id)
-        latest = {}
-        for status in statuses:
-            if status.context not in latest:
-                latest[status.context] = status.state
-
-        return all(status == "success" for status in latest.itervalues())
+    def status_checks_pass(self, pr_id, exclude=None):
+        if exclude is None:
```

Comment:
The only major difference with my version in #20 is that I call a different API endpoint to get summary status directly from GitHub if we're not excluding any contexts.